flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -173,7 +173,7 @@ class JobManager(
    * to run in the actor system of the associated job manager.
    */
   val webMonitorPort : Int = flinkConfiguration.getInteger(
-    ConfigConstants.JOB_MANAGER_WEB_PORT_KEY, -1)
+    JobManagerOptions.WEB_PORT.key(), -1)
```

Comment:
I am not sure, but why don't you use `flinkConfiguration.getInteger(JobManagerOptions.WEB_PORT)`? It would have different semantics (it would return `8081` instead of `-1` in case of missing value).